src/adafruit_blinka/microcontroller/generic_linux/spi.py

```diff
@@ -129,7 +129,7 @@ def write_readinto(
             self._spi.bits_per_word = self.bits
             data = self._spi.transfer(list(buffer_out[out_start : out_end + 1]))
             for i in range((in_end - in_start)):
-                buffer_in[i + in_start] = data[i]
+                buffer_in[i + in_start] = data[i + 1]
```

Comment:
The docs changes look good to me.

Is this `+ 1` intentional? If so this PR  LGTM.

Reply:
Sort of. That wasn't supposed to be part of this PR. It was for another issue that I started to test a while ago and forgot to revert. I'll remove that for now.